app/views/track.js

```diff
@@ -1,11 +1,51 @@
 import Mn from 'backbone.marionette';
+import application from '../application';
+import { timeToString } from '../libs/utils';
+
 
 const TrackView = Mn.ItemView.extend({
-    template: require('views/templates/track'),
+    
+    template: require('./templates/track'),
+
+    tagName: 'tr',
+
+    events: {
+        'click': 'play',
+        'click .delete': 'delete'
+    },
+
+    modelEvents: { change: 'render' },
+
+    play: function (e) {
+        application.upNext.setAttr('currentTrack', this.model);
+    },
+    
+    delete: function (e) {
+        console.log('delete')
+        const item = this.model;
+        item.set('hidden', true);
+        item.save();
+        application.allTracks.remove(item);
+        application.upNext.remove(item);
```

Comment:
Glitch: you should not call data management from a view. Instead, bind a listener in your `Tracks` collection that listen to the `change:hidden` event to remove the model from itself when it occurs.